ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -208,7 +208,7 @@ private String buildCorsRegexFromConfig() {
       return "";
     }
     if (allowedCorsOrigins.contains("*")) {
-      return "*";
+      return ".*";
```

Comment:
Is this a new format or something like that?

Reply:
yes, reported on Besu https://github.com/hyperledger/besu/pull/3296/files. Thats why I added new integration test case around it as well.